llama_stack/providers/remote/inference/vllm/vllm.py

```diff
@@ -148,19 +148,31 @@ async def _process_vllm_chat_completion_stream_response(
     async for chunk in stream:
         choice = chunk.choices[0]
         if choice.finish_reason:
-            yield ChatCompletionResponseStreamChunk(
-                event=ChatCompletionResponseEvent(
-                    event_type=event_type,
-                    delta=ToolCallDelta(
-                        tool_call=ToolCall(
-                            call_id=tool_call_buf.call_id,
-                            tool_name=tool_call_buf.tool_name,
-                            arguments=json.loads(tool_call_buf.arguments),
+            try:
+                args = {} if not tool_call_buf.arguments else json.loads(tool_call_buf.arguments)
```

Comment:
can the try-catch be only around `json.loads` so other errors aren't falsely suppressed?

Reply:
Sure. Updated